<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Filtered</string>
   </property>
   <widget class="QCheckBox" name="cb_dns_q4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IPV4 DNS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_dns_q6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IPV6 DNS</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>371</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Time Range</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>From : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>To    :</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dTimeEdit_From">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd AP h:mm:ss</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dTimeEdit_To">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>yyyy-MM-dd AP h:mm:ss</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>321</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_file_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>없음</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>530</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
